### PR TITLE
added scrapers for 2 more websites

### PR DESCRIPTION
**Pull request by pasha009**
I copied the format in the main `national_holidays.py`. I don't know if it's better to add options to fetch from a specified website. So for now the code is in separate files with some repeating part.

### CODE REVIEW

**Comment by pasha009** [comment]
I mistakenly thought that this repo is only for calendar website scrapers 😅. Please ignore previous comment.

**Comment by sakethramanujam** [comment]
Hello 👋 @pasha009 ! Thanks for your contribution. I suggest that you add them both into a folder with appropriate docs so that it would be easier to use. Let me know incase you need any other inputs. I will test the code tonight and provide comments.
 💯

**Comment by pasha009** [comment]
Thanks for showing interest 😄. I am not sure about what stuff to add in the docs. Could you help me by giving some examples of things to put in.

**Comment by sakethramanujam** [comment]
description about what the scraper does, I mean what website it scrapes and the output data format should be good for starters.

File: holidays/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Holidays
+
+scrap websites to generate holiday data in csv format
```
Comment (sakethramanujam) [inline holidays/README.md]: watch out
[scrape](https://www.merriam-webster.com/dictionary/scrape)
[scrap](https://www.merriam-webster.com/dictionary/scrap)

File: holidays/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+1. `panchang.py` scraps https://panchang.astrosage.com/calendars/indiancalendar
+2. `timeanddate.py` scraps https://www.timeanddate.com/holidays/india/
```
Comment (sakethramanujam) [inline holidays/README.md]: should we have a table like format? like say

|script|website scraped|
|:-|:-|
|something|[abc](something.com)|

**Review by sakethramanujam — CHANGES_REQUESTED** [review]

File: holidays/README.md (hunk 2 above)
Comment (pasha009) [reply holidays/README.md]: Yes, a table makes sense

File: holidays/README.md (hunk 1 above)
Comment (pasha009) [reply holidays/README.md]: thanks 😅

**Review by sakethramanujam — APPROVED** [review]
LGTM!

**Pull request merged**